src/ifcopenshell-python/ifcopenshell/util/element.py

```diff
@@ -91,9 +100,9 @@ def get_properties(properties):
 def get_predefined_type(element):
     element_type = get_type(element)
     if element_type:
-        predefined_type = getattr(element_type, "PredefinedType", None)
+        predefined_type = getattr(element_type, "ElementType", getattr(element_type, "ProcessType", None))
         if predefined_type == "USERDEFINED" or not predefined_type:
-            predefined_type = getattr(element_type, "ElementType", getattr(element_type, "ProcessType", None))
+            predefined_type = getattr(element_type, "ElementType", None)
         if predefined_type and predefined_type != "NOTDEFINED":
```

Comment:
Is this change correct?

Reply:
Nope, I must've been working from an older version.